main_logic/core.py

```
@@ -54,8 +54,8 @@
 # Setup logger for this module
 logger = get_module_logger(__name__, "Main")
 
-# TTS 错误码中不应自动重试的类型（欠费 / 配额耗尽）
-NO_RETRY_TTS_CODES = {'API_ARREARS', 'API_QUOTA_TIME'}
+# TTS 错误码中不应自动重试的类型（欠费 / 配额耗尽 / API Key 无效）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)